pkg/aws/policies.go

```diff
@@ -674,14 +676,75 @@ func parsePolicyDocument(path string) (PolicyDocument, error) {
 	return doc, nil
 }
 
+func isOCMRole(roleName *string) bool {
+	return strings.Contains(aws.StringValue(roleName), OCMRoleInfix)
+}
+
+func (c *awsClient) ListOCMRoles() ([]Role, error) {
+	var ocmRoles []Role
+	roles, err := c.ListRoles()
+	if err != nil {
+		return ocmRoles, err
+	}
+
+	for _, role := range roles {
+		if isOCMRole(role.RoleName) {
+			var ocmRole Role
+			ocmRole.RoleName = aws.StringValue(role.RoleName)
+			ocmRole.RoleARN = aws.StringValue(role.Arn)
+
+			policies, err := c.listPolicies(role)
```

Comment:
We are not listing policies why do we need this ?

Reply:
I've seen that we list policies in `rosa list account-roles`, I didn't realize it's redundant in `ocm-roles`.
I'll remove this line.